src/lines/LineSegmentsGeometry.ts

```diff
@@ -192,15 +192,9 @@ class LineSegmentsGeometry extends InstancedBufferGeometry {
     }
   }
 
-  toJSON = (): void => {
+  public toJSON = (): void => {
     // todo
   }
-
-  applyMatrix = (matrix: Matrix4): this => {
```

Comment:
this is seemingly deprecated and not in the three-ts-types file here https://github.com/three-types/three-ts-types/blob/master/types/three/examples/jsm/lines/LineSegmentsGeometry.d.ts, so i figure safe to just remove.